frontend/class-opengraph-image.php

```diff
@@ -317,6 +317,41 @@ protected function add_image_by_url( $url ) {
 		$this->add_image( array( 'url' => $url ) );
 	}
 
+	/**
+	 * Adds the possibility to short-circuit all the optimal variation logic with
+	 * your own size.
+	 *
+	 * @param int $attachment_id The attachment ID that is used.
+	 *
+	 * @return bool Whether the size has been overridden.
+	 */
+	protected function override_size( $attachment_id ) {
+		/**
+		 * Filter: 'wpseo_opengraph_image_size' - Allow overriding the image size used
+		 * for OpenGraph sharing. If this filter is used the defined size will always be
+		 * used for the og:image. The image will still be rejected if it is too small.
+		 *
+		 * Only use this filter if you manually want to determine the best image size
+		 * for the `og:image` tag.
+		 *
+		 * Use the `wpseo_image_sizes` filter if you want to use our logic. That filter
+		 * can be used to add an image size we will consider.
```

Comment:
What do you mean with `we will consider`? I suggest saying `That filter can be used to add an image size that needs to be taken into consideration within our own logic`